lib/chef-dk/cookbook_omnifetch.rb

```diff
@@ -18,13 +18,17 @@
 require 'cookbook-omnifetch'
 require 'chef-dk/shell_out'
 require 'chef-dk/cookbook_metadata'
+require 'chef-dk/helpers'
 
 require 'chef/http/simple'
 
 # Configure CookbookOmnifetch's dependency injection settings to use our classes and config.
 CookbookOmnifetch.configure do |c|
-  c.cache_path = File.expand_path('~/.chefdk/cache')
-  c.storage_path = Pathname.new(File.expand_path('~/.chefdk/cache/cookbooks'))
+  helpers = Class.new do
```

Comment:
If the output would only reasonably change during tests, then we could have an `@api private` method like `reset!`.